tools/text_processing/text_processing/macros.xml

```diff
@@ -6,7 +6,7 @@
         </requirements>
     </xml>
     <token name="@TOOL_VERSION@">9.5</token>
-    <token name="@VERSION_SUFFIX@">0</token>
+    <token name="@VERSION_SUFFIX@">1</token>
```

Comment:
This will bump the entire suite, and in return will bump all workflows using any of the tools in the suite. Is it not possible to override @VERSION_SUFFIX@ in the tool ?